app/android/app/src/main/AndroidManifest.xml

```diff
@@ -16,7 +16,7 @@
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
         android:launchMode="singleTask"
-        android:windowSoftInputMode="adjustResize"
+        android:windowSoftInputMode="adjustPan"
```

Comment:
This is what fixes the issue with text inputs continually losing focus on Android.

I've read the Android docs and don't feel like I entirely understand this setting if I'm honest, but it's recommended by a number of people online e.g. https://stackoverflow.com/a/28537589/15422951 and I haven't been able to spot any adverse effects of changing it.

Reply:
The only thing the [documentation](https://developer.android.com/guide/topics/manifest/activity-element.html) warns about with adjustPan is that the soft keyboard could obscure elements that the user needs to type in.

That explains a common annoyance, I've seen that a lot where you need to swipe back after entering text to get at the "OK" or "Next" button and you're praying it doesn't go back a screen.

I don't think this is an issue either and its got to be less annoying than losing focus.

Reply:
Wee [video](https://www.youtube.com/watch?v=FOibPikr0qc) for anyone who wants to see the difference.

Reply:
Thanks @DougieWougie -- good to understand it better.  As an iPhone user I'm not used to these things! 🙂  Hopefully this will work for us as is.